event_files/rvv_retired.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "EventName": "RVV.ALL.RET",
+        "EventCode": "0x0",
+        "BriefDescription": "Number of RVV instructions retired",
+        "PublicDescription": "Number of RVV instructions retired"
+    },
```

Comment:
If we do the INST.*.RET thing suggested above, maybe these should be:
INST.RVV.RET
INST.RVV.INT.RET
INST.RVV.FP.RET

The rest are element events, so they would be unchanged.

Reply:
Changed